wiki/Ranking_Criteria/en.md

```diff
@@ -139,12 +139,12 @@ Glossary
 
 ### Rules
 #### Technical
-- **Metadata must be consistent across all difficulties of a beatmap set.**
-- **Guest mappers, storyboarders, skinners and hitsounders must be added to the tags of a beatmap set.** This is to give credit where credit is due and help others identify the main contributors of any given beatmap set. Usernames containing whitespaces should be written with underscores instead.
+- **Metadata must be consistent across all difficulties of a beatmapset.**
+- **Guest mappers, storyboarders, skinners and hitsounders must be added to the tags of a beatmapset.** This is to give credit where credit is due and help others identify the main contributors of any given beatmapset. Usernames containing single characters separated by whitespaces should replace whitespaces with underscores.
```

Comment:
```suggestion
- **Guest mappers, storyboarders, skinners and hitsounders must be added to the tags of a beatmapset.** This is to give credit where credit is due and help others identify the main contributors of any given beatmapset. Usernames containing single characters separated by whitespaces must have the whitespaces replaced with underscores.
```
usernames don't make decisions, people do

Reply:
clayton makes the right decisions